### PR TITLE
Migration from xproj to csproj

### PR DESCRIPTION
**Pull request by SlyNet**
Update to MvvmCross 5 and migration to SDK csproj format

### CODE REVIEW

File: src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <Company>Autofac Project - http://autofac.org</Company>
+    <Product>Autofac</Product>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseUrl></PackageLicenseUrl>
+    <Copyright>Copyright © 2017 Autofac Contributors</Copyright>
+    <Description>MvvmCross integration for Autofac IoC</Description>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="autofac" Version="4.6.1" />
+    <PackageReference Include="MvvmCross.Core" Version="5.1.1" />
```
Comment (tillig) [inline src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj:17]: Why the update to MvvmCross 5? Did you make changes that aren't compatible with 4 and would force consumers to take an upgrade?

File: src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <Company>Autofac Project - http://autofac.org</Company>
+    <Product>Autofac</Product>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseUrl></PackageLicenseUrl>
+    <Copyright>Copyright © 2017 Autofac Contributors</Copyright>
+    <Description>MvvmCross integration for Autofac IoC</Description>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="autofac" Version="4.6.1" />
```
Comment (tillig) [inline src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj:16]: "Autofac" should be capitalized to match the actual name of the package. Also, this should be version 4.0.0 to enable wider compatibility unless there are code changes made here that require consumers take an update.

File: src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81</PackageTargetFallback>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <Company>Autofac Project - http://autofac.org</Company>
+    <Product>Autofac</Product>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseUrl></PackageLicenseUrl>
+    <Copyright>Copyright © 2017 Autofac Contributors</Copyright>
```
Comment (tillig) [inline src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj:12]: The copyright from the original project was 2014.

**Comment by tillig** [comment]
I've added a few comments inline but there's a lot missing here:

- Package icon
- License URL
- Tags
- Code analyzers (FxCop / StyleCop)

It also looks like it'll require folks to take updates to both MvvmCross and Autofac but I don't see any code changes that were made that would force that requirement.

[Here's the .csproj from Autofac.Extensions.DependencyInjection.](https://github.com/autofac/Autofac.Extensions.DependencyInjection/blob/develop/src/Autofac.Extensions.DependencyInjection/Autofac.Extensions.DependencyInjection.csproj) It might give you an idea of what we'd be looking for in an update to the project format.

**Comment by SlyNet** [comment]
Thank you for reviewing the request. You are right, there are no changes to implementation, just dependency update. I should've thought about it before doing the request :)

**Pull request closed without merging**